File: Matemática/1585/1585.cpp
#include <iostream>

using namespace std;

int main()
{
    int qtdTestes;
    cin >> qtdTestes;
    
    while(qtdTestes--) {
        short x, y;
        cin >> x >> y;
        
        cout << (x*y) / 2 << " cm2\n";
    }
    return 0;
}

File: Matemática/1585/1585.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		short casosDeTeste = scan.nextShort();
		
		while(casosDeTeste-- > 0) System.out.printf("%d cm2\n", (scan.nextByte() * scan.nextByte()) / 2);
	}
}